### ISSUE TITLE
[BUG] There are a mixture of International English and US English words in list

### ISSUE THREAD

**Issue by roubaobaozi** [nvlgzr/fngrng#6]
# Describe the bug

There are a mixture of International English and US English words in list

For example, International (colour, customise) and US (color, customize)

There are also non-words in the list, eg. `devel` and `ch` or company/brand names eg. `cisco`

- [X] I have verified that this issue hasn't yet [been reported](https://github.com/nvlgzr/fngrng/issues?q=is%3Aissue+label%3Abug)

# Steps To Reproduce
1. Select all words
2. Either run a long enough test to come across words from each, or check bundle.js for the wordlist

## System Details
 - OS: macOS 11.6.2
 - Browser: Brave 1.35.103 (Chromium: 98.0.4758.102 (Official Build))

**Comment by nvlgzr** [comment]
Thanks for the feedback, @roubaobaozi!

This can go in several directions, hence the additional labels I just added ↑.

1. The minimal response to the issue would be to cut one dialect out of the product, along with non-words and proper nouns
2. A mid-level effort would include multiple lists, allowing selection between them.
3. The "correct" response to the issue would be to implement app-wide i18n.

Perhaps you could help me pick a path and implement it?

### What I'm thinking so far

Option 3 is way beyond anything I've got the time and energy to tackle alone. For starters, I lack deep i18n expertise, having only rarely worked on multi-lingual projects. I also lack expertise in almost _all_ keyboard variations that folks might want fngrng to support. This doesn't mean I'm against fully internationalizing _fngrng_, but it does mean that I can't really begin to consider this without some serious collaboration.

Between 1 & 2, I prefer option 2 as it's not a whole lot more effort, and has the benefit of not favoring either of these two predominant flavors of English, while also laying the ground work for additional word list variants. The implementation should be relatively simple, since there's no change implied to anything but the word list and some trivial UX components.

The main obstacle is, once again, my own lack of expertise. I don't trust myself to correctly edit the current wordlist (inherited from [colemakclub](https://github.com/gnusenpai/colemakclub), which I believe in turn pulled it from [colemak.academy](https://colemak.academy)…whose creator is anonymous).

### Long story short

In order for this issue to move forward, folks with more expertise than me will need to chime in with clear, evidence-backed advice on how to select and/or generate the wordlist(s) to replace the current one.

**You interested?** Comments welcome!
